docs/asyncapi.yaml

```diff
@@ -0,0 +1,126 @@
+asyncapi: 3.0.0
+info:
+  title: API contract book for Single Front Door user communications
```

Comment:
Can probably simplify the title as by being a spec doc it's clear it's an API contract book.  I'd say something like Single Front Door messages.